app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -625,11 +625,11 @@ def parse_container(container, pod_id)
       }
       state_attributes = parse_container_state container.lastState
       state_attributes.each { |key, val| h[key.to_s.prepend('last_').to_sym] = val } if state_attributes
-      h.merge! parse_container_state container.state
+      h.merge! parse_container_state container.state.to_h
```

Comment:
Also, please consider to make this more readable:

    h.merge!(parse_container_state(container.state))